.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: Report a problem using F´
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+|**Field**|**Value**|
+|:---|:---|
+|**_Report Date_**| [DATE] |
+|**_Reported By_**| [USER] |
+|**_Branch/Hash_**| |
+|**_Short Description_**| |
```

Comment:
What's the difference between the title and the short description? Do we need both? I feel like when I'm filing this out I'd just copy and paste in the title.